Add recent activity widget back with /roles

danielcolinjames opened the pull request:
![Screen Shot 2022-01-24 at 9 58 06 AM](https://user-images.githubusercontent.com/10648471/150838572-13fc6e57-9942-4740-b412-8e55972ef06a.png)

danielcolinjames committed:
Add recent activity widget back with /roles

tyler-whitman commented on components/Widgets/RecentActivityWidget.js:
```diff
@@ -41,9 +41,10 @@ const RecentActivityWidget = ({ context, address }) => {
           ) : (
             transactions.map((t) => (
               <div className="flex items-center justify-between w-full">
+                <div className='w-2 h-4 mr-2 rounded-full' style={{backgroundColor: t.type === 'poc_receipts_v1' ?  getTxnTypeColor(getPocReceiptRole(t.role)) : getTxnTypeColor(t.type)}} />
```
Any concerns for testnet with `poc_receipts_v2`?

danielcolinjames replied:
good catch!

danielcolinjames committed:
Merge branch 'v2' into recent-activity-v2

danielcolinjames committed:
Merge branch 'v2' into recent-activity-v2

danielcolinjames committed:
Recent activity widget fixes
- styling improvement for color slice
- recognize different roles within poc_receipt
